.eslintrc

```diff
@@ -1,16 +1,29 @@
 ---
   extends: airbnb/legacy
+  env:
+    node: true
+    mocha: true
```

Comment:
Enabling mocha globally has unintended side effects, e.g. it registering some globals that we do not want. Please disable it.